CHANGELOG.md

```diff
@@ -2,7 +2,23 @@ Changelog
 =========
 # (unreleased)
 
-# 0.4.7
+# 0.4.8 (2018-10-20)
+
+- **Coordinator**: Fix bugs in database creation API (error not displayed in logs and provided number of shards not honored)
```

Comment:
Since this is a fix, can you format as (to be consistent with other changelogs):
```md
- FIX **Coordinator**: 
```